### ISSUE TITLE
Constrain camera movements to stay close to the map

### ISSUE THREAD

**Issue by TimJentzsch** [Leafwing-Studios/Emergence#115]
Currently, you can zoom out infinitely and pan away infinitely, so you might get lost and be unable to find the map again if you're moving away too far.

At some point we should restrict the camera movements to stay within a reasonable distance of the map with a capped in and out zoom range.

**Comment by alice-i-cecile** [comment]
Agreed! These are probably best done separately: capping zoom will be very easy.
